sdk/security/keyvault/azsecrets/CHANGELOG.md

```diff
@@ -1,6 +1,16 @@
 # Release History
 
-## 1.0.0 (2023-05-11)
+## 1.0.0 (Unreleased)
+
+### Breaking Changes
+* Renamed `Client.ListSecrets` to `Client.ListSecretProperties`
+* Renamed `Client.ListSecretVersions` to `Client.ListSecretPropertiesVersions`
+* Renamed `SecretBundle` to `Secret`
+* Renamed `DeletedSecretBundle` to `DeletedSecret`
+* Renamed `SecretItem` to `SecretProperties`
+* Renamed `DeletedSecretItem` to `DeletedSecretProperties`
+* Renamed `Kid` to `KeyID`
```

Comment:
I think we want `KID` instead. Key Vault represents keys as JWKs, in which "kid" is the correct name.